### PR TITLE
Remove `ViewConsentManager` dependency for admin api scope

### PR DESCRIPTION
**Pull request by linh-transcend**
## Related Issues

- https://github.com/transcend-io/main/pull/23125#discussion_r1253511133

## Security Implications

_[none]_

## System Availability

_[none]_

### CODE REVIEW

File: src/scopes.ts [hunk 1]
```diff
@@ -641,7 +641,7 @@ const SCOPES_WITHOUT_VIEW_ONLY: {
   },
   [ScopeName.ViewManagedConsentDatabaseAdminApi]: {
     title: 'View Managed Consent Database Admin Api',
-    dependencies: [ScopeName.ViewConsentManager],
+    dependencies: [],
```
Comment (anotherminh) [inline src/scopes.ts:644]: @linh-transcend i think you also want to make the viewConsentManager depends on this new scope? per mike's suggestion?

**Review by anotherminh — DISMISSED** [review]

File: src/scopes.ts (hunk 1 above)
Comment (linh-transcend) [reply src/scopes.ts:647]: my bad. just added!

**Comment by linh-transcend** [comment]
@anotherminh do i need all the pre-commits to pass to merge? Can't figure what's wrong with this?

https://github.com/transcend-io/privacy-types/actions/runs/5468283802/jobs/9955704109?pr=102

**Comment by anotherminh** [comment]
@linh-transcend lmao that error... yea you want it to pass. I'm guessing there is a weird hwitespace issue or something

**Comment by anotherminh** [comment]
there's a way to run the pre-commit locally as well

**Review by anotherminh — APPROVED** [review]

**Pull request merged**